config/routes.rb

```diff
@@ -15,7 +15,7 @@
   SamlEndpoint.suffixes.each do |suffix|
     get "/api/saml/metadata#{suffix}" => 'saml_idp#metadata', format: false
     match "/api/saml/logout#{suffix}" => 'saml_idp#logout', via: %i[get post delete]
-    match "/api/saml/remotelogout#{suffix}" => 'saml_idp#remotelogout', via: %i[get post delete]
+    post "/api/saml/remotelogout#{suffix}" => 'saml_idp#remotelogout'
```

Comment:
Removed the other actions since we really only support the POST binding and not the Redirect binding.